boards/DCM/Inc/App/configs/App_TorqueRequestThresholds.h

```diff
@@ -7,3 +7,7 @@
  * @see Setting up the Controller to run with Emrax Motor Rev 1.2 PDF pg. 6
  */
 #define MAX_TORQUE_REQUEST_NM 132.0f;
```

Comment:
This is slightly sketch, can we change to 50.0f for now and increase as we validate?

Reply:
Done, see [PR 753](https://github.com/UBCFormulaElectric/Consolidated-Firmware/pull/753)